compute_tools/src/config.rs

```diff
@@ -75,7 +75,7 @@ pub fn write_postgres_conf(
         neon_safekeepers_value.push_str(&spec.safekeeper_connstrings.join(","));
         writeln!(
             file,
-            "neon.safekeepers={}",
+            "neon.safekeeper_connstrings={}",
```

Comment:
Seems ok to me thanks to comma (`,`) being a reserved character in percent-encoding scheme used in Postgres connection strings, as documented at https://datatracker.ietf.org/doc/html/rfc3986#section-2.1. It might be worth saying so in the comments in that part of the code?